### DISCUSSION TITLE
dev compilation fail

### DISCUSSION THREAD

**Discussion by fnc12** [fnc12/sqlite_orm#857] [General]
@trueqbit compilation fails on mac with clang. I need your assistance for fixing. Thanks
<img width="1440" alt="Снимок экрана 2021-11-30 в 18 30 43" src="https://user-images.githubusercontent.com/6009640/144077027-25bbc386-6b85-468d-bc3a-5099c926c8c8.png">

**Comment by fnc12** [comment]
So I created `dev-with-mac-to-fix` branch from current `dev` and dropped commits that failed compilation. IDK how to reach you. Please fix your feature clang compilation. Thanks

**Reply by trueqbit** [reply]
Commit messages show the author ;) Sorry, I didn't see your previous message, I was AFK.
I'll take a look - which clang version is it that you are using on macos?

**Reply by fnc12** [reply]
I guess you can use any version

**Reply by fnc12** [reply]
my local version is 13.0.0

**Comment by trueqbit** [comment]
@fnc12 Shall I just create another pull request from FireDaemon:pointer_arg_validation -> fnc12:dev?

**Reply by fnc12** [reply]
Better take `dev-with-mac-to-fix` and try to fix its compilation. `dev-with-mac-to-fix` contains all your commits

**Reply by trueqbit** [reply]
Fixed with PR #860.
